package.json

```diff
@@ -85,16 +84,15 @@
     "glob": "10.3.4",
     "graphql": "16.8.0",
     "helmet": "7.0.0",
-    "jsdom": "22.1.0",
     "jsonwebtoken": "8.5.1",
```

Comment:
Can we remove jsonwebtoken? We currently have trouble upgrading it, so good riddance.

Reply:
With this change its only used in the cypress test, I will see about an alternative.